pwiz_tools/Skyline/CommandLine.cs

```diff
@@ -513,6 +518,23 @@ private bool ImportAnnotations(CommandArgs commandArgs)
             }
         }
 
+        private bool ImportPeakBoundaries (CommandArgs commandArgs)
+        {
+            try
+            {
+                long lineCount = Helpers.CountLinesInFile(commandArgs.ImportPeakBoundaries);

```

Comment:
Needs something like this:
_out.WriteLine("Importing peak boundaries from {0}, Path.GetFileName(commandArgs.ImportPeakBoundariesPath));